### ISSUE TITLE
再接続試行の間隔を広げる

### ISSUE THREAD

**Issue by vorj** [fixstars/sacana#24]
Slackとの接続が切れたときのリトライ間隔があまり長くないため、瞬断でない場合は大抵リトライのしすぎでsystemdから怒られてデーモンが止まってしまうので、もう少しのんびりさせたほうが良さそう

**Comment by toru-fukaya** [comment]
そうね。ただ、プログラムとしては死んで、起動しての繰り返しをしているはずなので、起動してからしばらく待つか死ぬ前にしばらく待つかになりそう? (前者の場合は普通の起動も遅くなってしまいそう)

なんか異常終了する前にファイルを作っておいて(vimのswpファイルみたいな感じで)そのファイルがあったらしばらく待つとかならいい感じになるかも

**Comment by vorj** [comment]
> なんか異常終了する前にファイルを作っておいて(vimのswpファイルみたいな感じで)そのファイルがあったらしばらく待つとかならいい感じになるかも

そうですね、可能ならば異常終了を繰り返す度に待ち時間を徐々に伸ばす仕組みが好ましいとは思います(が、そこまで作り込む必要があるかはまた考えたほうが良さそう)
